front/src/applications/operationalStudies/types.ts

```diff
@@ -154,3 +156,23 @@ export type StudyCardDetails = SearchResultItemStudy | StudyWithScenarios;
 export type ScenarioCardDetails = SearchResultItemScenario | ScenarioWithDetails;
 
 export type CategoryColors = { normal: string; hovered: string; background: string };
+
+export type PathProjectionResult = {
+  path: PathItem[] | undefined;
```

Comment:
Looks like `path` is always defined now
```suggestion
  path: PathItem[];
```